packages/styles/src/components/form-check.scss

```diff
@@ -109,7 +109,10 @@
 
   &-label {
     flex: 1;
-    padding-inline-start: form-check.$form-check-column-gap;
+
+    :not(.form-switch) > & {
```

Comment:
```suggestion
    :not(.form-switch, .radio-button-card, .checkbox-button-card) > & {
```

Reply:
It's becoming a bit convoluted, but at least the intend is clear. Maybe we should not have a `.form-check` that is shared with `.form-switch`